### DISCUSSION TITLE
SSR example

### DISCUSSION THREAD

**Discussion by jamauro** [spinspire/pocketbase-sveltekit-starter#13] [Ideas]
Thanks for putting this repo together and sharing it!

It would be interesting to see some best practices around SSR. I'm particularly interested in realtime subscriptions with SSR. Still learning how all the pieces fit together.

**Comment by jkdoshi** [comment]
At least to me, SSR and  realtime subscriptions are contradictory. SSR means generating HTML on the server, while realtime subscriptions make sense only on the browser.

Also, this approach focuses on JAMStack instead of SSR.

**Reply by jamauro** [reply]
Yeah, I can see what you mean. I saw an approach where the first set of results was SSR generated and then realtime subscriptions took over in the browser. I suppose that would work but just wondering if there were other ideas.
